docs/source/user-guide/general/ci_cd.rst

```diff
@@ -0,0 +1,121 @@
+=================================
+Using Deepchecks In CI/CD
+=================================
+
+CI/CD is a practice of automating steps in a software life cycle such as testing, and deploying. The same practice of
+traditional software development can be applied to machine learning, which allows for frequent and automated testing
+and validation of models, which reduce the risk of errors and improve the overall quality of the model.
+
+For example, CI/CD in machine learning can be used in scenarios such as:
+
+* Data integrity checks: Before the model training process, it’s important to validate the integrity of the data used
+  for training. This can include checks such as data completeness, missing values, and data type consistency.
+* Model training: The model is trained on the validated data set.
```

Comment:
I would greatly the part about validating train test split as it is our strength,
Normally to avoid leakage, the test will be of more current data so it may very well be drifted from the training set.
for Periodic retraining of a model this can also mean compering the current training batch to the previous training batch

Reply:
I would rename this stage to Model Validation - and elaborate about what should be validated? give links to different checks and suites as examples.

Reply:
what? you would rename model training "model validation"?